### PR TITLE
Cookbook block

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

**Comment by raphaelcarles** [comment]
qq coquilles : 
- This cookbook article covers the created of block call `Hello` -> This cookbook article covers the creation of block called `Hello`.
- it returns only rendering the template twig -> it returns only the rendering of the the template twig
- For it, it must created a class which extend ... and added the necessary field as we would for a -> For it, it must create a class which extend ... and add the necessary field as we would for a
- It must created a class which extend  -> It must create a class which extend 

sinon :+1:

File: en/cookbook/create_block.rst [hunk 1]
```diff
@@ -0,0 +1,412 @@
+How to Create a Block
+=====================
+
+In Open Orchestra, pages are made of blocks representing data to display.
+
+The Block creation `section`_ covers the strategies needed to a block.
```
Comment (veolys-nanne) [inline en/cookbook/create_block.rst:6]: I'm not sure but I would say needed for a block

File: en/cookbook/create_block.rst [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+
+This cookbook article covers the created of block call ``Hello``.
+
+
+Front Office display strategy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, must be created the block display strategy of Front Office in ``FrontBundle``.
```
Comment (veolys-nanne) [inline en/cookbook/create_block.rst:14]: First, the block display strategy must be created in ....

File: en/cookbook/create_block.rst [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+In our case the strategy is very simple, it returns only rendering
```
Comment (veolys-nanne) [inline en/cookbook/create_block.rst:15]: . It only returns the render of...

File: en/cookbook/create_block.rst [hunk 4]
```diff
@@ -0,0 +1,412 @@
...
+the template twig ``Hello:show.html.twig`` with a parameter name.
+The name parameter is a parameter of the block that can be contributed
+in the Back Office as we shall see later.
```
Comment (veolys-nanne) [inline en/cookbook/create_block.rst:18]: we will see later

File: en/cookbook/create_block.rst [hunk 5]
```diff
@@ -0,0 +1,412 @@
...
+    {# src/Acme/Bundle/FrontBundle/Resources/views/Block/Hello/show.html.twig #}
+
+    {% extends 'OpenOrchestraDisplayBundle:Block:layout.html.twig' %}
+
+    {% block content %}
+        Hello {{ name }} !
+    {% endblock %}
+
+Once our template and our strategy created, it is necessary to register strategy as a service tagged by
+``open_orchestra_display.display_block.strategy`` for it to be found by the system.
```
Comment (veolys-nanne) [inline en/cookbook/create_block.rst:99]: for it to be found by the system => to be found by the system

**Pull request closed without merging**